Dockerfile.rhel7

```diff
@@ -6,7 +6,7 @@ RUN make build --warn-undefined-variables
 
 FROM registry.svc.ci.openshift.org/ocp/4.4:base
 RUN mkdir -p /usr/share/bootkube/manifests /manifests
-COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/* /usr/share/bootkube/manifests/
+COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/manifests /usr/share/bootkube/
 COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/cluster-kube-apiserver-operator /usr/bin/
 COPY manifests/*.yaml /manifests/
```

Comment:
I think that those two lines could be simplified to just a single command `COPY manifests /manifests`
```
COPY manifests/*.yaml /manifests/
COPY manifests/image-references /manifests/
```

Reply:
that's a good observation, we are not storing anything else in there